Lorem Ipsum XR Translator/Assets/Scripts/SpeechToText/AzureSpeechToTextButton.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+using Microsoft.CognitiveServices.Speech;
+using Microsoft.MixedReality.Toolkit.UI;
+using TMPro;
+using Config;
+
+public class AzureSpeechToTextButton : MonoBehaviour
+{
+    private string serviceRegion = "northeurope";
+
+    public ToolTip toolTip;
+
+    public async void OnButtonPressed()
+    {
+        Debug.Log("Button was pressed.");
+
+        SpeechConfig config = SpeechConfig.FromSubscription(Secrets.GetSTTSpeechSDKKey(), serviceRegion);
+        using (var recognizer = new SpeechRecognizer(config))
```

Comment:
Personal preference, but would like to see 'var' replaced with strongly-typed types when not dealing with generics. So in this case the recognizer should be a SpeechRecognizer object.